conformance/utils/suite/suite.go

```diff
@@ -177,6 +190,12 @@ func (test *ConformanceTest) Run(t *testing.T, suite *ConformanceTestSuite) {
 		}
 	}
 
+	for _, feature := range test.MinChannel {
+		if Experimental < Standard {
+			t.Skip("Skipping %s: suite does not support %s", test.ShortName, feature)
+		}
+	}
```

Comment:
I think we want this to look more like this:
```suggestion
	if test.MinChannel < suite.MinChannel {
		t.Skipf("Skipping %s: only testing %s channel", test.ShortName, suite.MinChannel)
	}
```

Reply:
Just made this one too